yt_dlp/extractor/prankcast.py

```diff
@@ -0,0 +1,71 @@
+from .common import InfoExtractor
+from ..utils import parse_iso8601
+
+
+class PrankCastIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?prankcast\.com/.*/showreel/(?P<id>\d+)-(?P<display_id>.+)'
+    _TESTS = [{
+        'url': 'https://prankcast.com/Devonanustart/showreel/1561-Beverly-is-back-like-a-heart-attack-',
+        'info_dict': {
+            'id': '1561',
+            'ext': 'mp3',
+            'title': 'Beverly is back like a heart attack!',
+            'uploader': 'Devonanustart',
+            'upload_date': '20220825'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id, display_id = self._match_valid_url(url).group('id', 'display_id')
+
+        webpage = self._download_webpage(url, video_id)
+
+        # Extract the JSON
+        json_info = self._search_nextjs_data(webpage, video_id)['props']['pageProps']['ssr_data_showreel']
+
+        # Get the broadcast URL and the recording hash.
+        # The full URL is {broadcast_url}/{recording_hash}.mp3
+        broadcast_url = json_info.get('broadcast_url')
+        recording_hash = json_info.get('recording_hash')
+        url = broadcast_url + recording_hash + ".mp3"
+
+        # Get author (AKA show host)
+        uploader = json_info.get('user_name')
+
+        # Get the co-hosts/guests
+        if uploader != '':
+            guests = [uploader]
+        else:
+            guests = []
```

Comment:
This can be rewrite as:
```suggestion
            guests = [] if not uploader else [uploader]
```
probably, you can use list comperhension for guest value